proposals/xxxx-package-manager-manifest-api-redesign.md

```diff
@@ -0,0 +1,377 @@
+# Package Manager Manifest API Redesign
+
+* Proposal: [SE-XXXX](xxxx-package-manager-manifest-api-redesign.md)
+* Author: [Ankit Aggarwal](https://github.com/aciidb0mb3r)
+* Review Manager: TBD
+* Status: **Discussion**
+
+## Introduction
+
+This proposal is a redesign of `Package.swift` manifest APIs provided by Swift Package Manager.  
+Note: It does not aim to change any actual underlying functionality.
+
+## Motivation
+
+`Package.swift` manifest API was designed prior to the [API Design Guidelines]
+(https://swift.org/documentation/api-design-guidelines/), and its design was not reviewed by the
+evolution process. We would like to redesign this API as necessary to provide a clean,
+conventions-compliant API that we can rely on in the future.
+
+## Proposed solution
+
+This is how a regular manifest looks like currently:
```

Comment:
how ... looks ... (no like)
or
  what a ... looks like ...

http://english.stackexchange.com/questions/55672/is-it-what-it-looks-like-or-how-it-looks-like